.github/workflows/ophan.yml

```diff
@@ -0,0 +1,57 @@
+name: "[Ophan] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 7AM, Mondays to Fridays
+    - cron: "0 7 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        run: |
+          REPOS=(
+            ophan
```

Comment:
As only one repository is monitored, should we use inline configuration for simplicity? See [guardian-engineering-site.yml](https://github.com/guardian/prnouncer-config/blob/760318c6771de45fce78d0bfed30e63c4923efd0/.github/workflows/guardian-engineering-site.yml#L26) for an example.